tools/source_updater/updater/handler.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import subprocess
+from typing import Set
+
+from airbyte_cdk.sources.declarative.manifest_declarative_source import ManifestDeclarativeSource
+from updater.catalog import CatalogMerger
+from updater.config import Config
+from updater.source import SourceRepository
+
+
+logger = logging.getLogger("handler")
+
+
+class SourceUpdaterHandler:
```

Comment:
Need to add tests around this class

Reply:
just double checking. i see tests so we're good?

Reply:
Yes, added those last week so I'll resolve this comment!